lib/ssl/src/ssl_internal.hrl

```diff
@@ -201,11 +201,17 @@
         }).
 
 -define(DEFAULT_DEPTH, 10).
-
+-define(DEFAULT_OCSP_STAPLING, false).
+-define(DEFAULT_OCSP_NONCE, true).
+-define(DEFAULT_OCSP_RESPONDER_CERTS, []).
+-define(GET_OPT_FROM_MAP(MapName, Opt, Default, SslOpts),
+        fun() ->
+            case maps:get(MapName, SslOpts, disabled) of
+                #{Opt := Value} ->
+                    Value;
+                _ ->
+                    Default
+            end
+        end()).
```

Comment:
Do we need this macro?  We should only need to look inside of the map when it exists.

Reply:
You're right. I will remove it.